src/main/java/entities/User.java

```diff
@@ -0,0 +1,65 @@
+package entities;
+
+public class User;
+    ArrayList<Post> favourites;
+        ArrayList<Post> posts;
+        boolean isAdmin;
+        String userName;
+        String password;
+
+public User(String name, String pass, boolean admin) {
+        this.favourites = ArrayList();
```

Comment:
Need to initialize an ArrayList with the new keyword for favourites and posts attributes.